notebooks/swarm.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    while True:
</pre><p>This is nice but can't we simply have current_agent here rather than: </p><p>current_agent_name = new_agent_name or current_agent_name</p><p>because you always return either current agent or other agent. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>thanks!</p><p> </p><p>I will simplify this...</p>